cmd/device-worker/main.go

```diff
@@ -122,6 +123,11 @@ func main() {
 
 	hw := hardware2.Hardware{}
 
+	GracefulRebootChannel = make(chan string, 10)
```

Comment:
Why is it 10?

Reply:
I thought that we might need it for another things later but I will change it to be 1 (the default)